WORKSPACE.bzlmod

```diff
@@ -0,0 +1,2 @@
+# This file marks the root of the Bazel workspace.
```

Comment:
i thought `WORKSPACE` marked the root of the Bazel workspace.  why do we need a second WORKSPACE file?

Reply:
This is for bzlmod, so that it replaces the existing WORKSPACE file when bzlmod is enabled, so it doesn't confuse the build system. 

See also https://docs.google.com/document/d/1JtXIVnXyFZ4bmbiBCr5gsTH4-opZAFf5DMMb-54kES0/edit#heading=h.y054fjub9max